bugwarrior/services/gitlab.py

```diff
@@ -70,6 +74,181 @@ def default_priorities(cls, values):
         return values
 
 
+class GitlabClient(ServiceClient):
+    """Abstraction of Gitlab API v4"""
+
+    def __init__(self, host, token, use_https, verify_ssl):
+        if use_https:
+            self.scheme = 'https'
+        else:
+            self.scheme = 'http'
+        self.verify_ssl = verify_ssl
+
+        self.host = host
+        self.token = token
+
+    def _base_url(self):
+        return f"{self.scheme}://{self.host}/api/v4/"
```

Comment:
sure. I basically made this equivalent to the implementation in the Github service, but I can move that.

Reply:
Actually, my motivation was so that we can easily test the different configurations, as https for instance is set in the constructor. Do you think that is worth it? Otherwise, I will move it back into `_fetch()`

Reply:
I'll defer to your judgement on this one.